server/package.json

```diff
@@ -69,7 +69,7 @@
     "compression": "^1.8.0",
     "cookie": "^1.0.2",
     "cookie-parser": "^1.4.7",
-    "cron": "^3.5.0",
+    "cron": "4.3.0",
```

Comment:
This needs to match *exactly* the version nestjs/schedule is using, so we cannot use 4.3.1 atm.